visualization/package.json

```diff
@@ -73,7 +53,7 @@
 		"e2e": "cross-env TEST_TYPE=\"e2e\" jest --config ./jest.config.json .*\\.e2e\\.ts --coverage false",
 		"e2e:auto": "npm run e2e -- --watch",
 		"e2e:ci": "npm run e2e -- --runInBand",
-		"package": "node -e 'fs.rmSync(\"dist/packages\", { recursive: true, force: true })' && build --concurrent --tasks win-x86,win-x64,linux-x86,linux-x64,mac-x64 --mirror https://dl.nwjs.io/ . && bestzip ./dist/packages/codecharta-visualization-$npm_package_version-web.zip ./dist/webpack",
```

Comment:
Is the removel controlled by the configuration in build-nwjs.js?
I am not sure which property is responsible for that.

Reply:
But I will re-add this clean up call, because the implementation inside nw-builder does not reliably remove unused data